src/runner_manager.py

```diff
@@ -264,7 +265,10 @@ def get_github_info(self) -> Iterator[RunnerInfo]:
             List of information from GitHub on runners.
         """
         remote_runners = self._get_runner_github_info()
-        return iter(RunnerInfo(runner.name, runner.status) for runner in remote_runners.values())
+        return iter(
+            RunnerInfo(runner.name, runner.status, runner.busy)
+            for runner in remote_runners.values()
+        )
```

Comment:
May I ask what the `iter` wrapper is for please?

Reply:
it should be possible to drop the `iter` and just return with `(...)` which creates a generator